repository/kafka/operator/operator.yaml

```diff
@@ -34,38 +48,35 @@ tasks:
         - file: /tmp/tls.crt
           kind: Secret
           key: certificate
-  - name: user-workload
-    kind: Apply
+  # feature tasks
+  - name: service-monitor
+    kind: Toggle
     spec:
+      parameter: ADD_SERVICE_MONITOR
```

Comment:
Should we have a common pattern for these toggle task parameter names? Like `*_TOGGLE` or smth.?

Reply:
I thought about that and kept it as they were in this PR and I was planning to do have a prefix like `ENABLE_*` in a follow-up PR. And mark that as a breaking change in the next PR if somebody is using any of these changed parameters. 

`ENABLE_*` because they are not only `toggle` task params as they might be used elsewhere, example is external advertised listener param that add resources(services) with toggle task feature but is also used to in statefulset to set the right advertised listener port. WDYT?

Reply:
SGTM! 👍